### PR TITLE
BA-1102 - Fix incorrect deadline in email notification

### PR DESCRIPTION
**Pull request by sutherlanda**
fix(opportunities): Fix incorrect deadline in email notification

- Use moment-timezone to format deadline
- Clean-up templates

Fixes BA-1102.

### CODE REVIEW

File: modules/core/server/controllers/CoreServerHelpers.ts [hunk 1]
```diff
@@ -158,19 +159,14 @@ class CoreServerHelpers {
...
-		const day = d.getDate();
-		return monthNames[monthIndex] + ' ' + day + ', ' + year;
-	};
 
-	public formatTime = d => {
-		return ('0' + d.getHours()).slice(-2) + ':' + ('0' + d.getMinutes()).slice(-2);
+		return moment(date)
+			.tz('America/Vancouver')
+			.format('MMMM Do, YYYY [at] HH:mm z');
 	};
```
Comment (SteveChapmanBCDX) [inline modules/core/server/controllers/CoreServerHelpers.ts]: This is a big improvement.

File: modules/opportunities/server/controllers/OpportunitiesServerController.ts [hunk 2]
```diff
@@ -836,15 +836,11 @@ class OpportunitiesServerController {
 			name: opportunity.name,
 			short: opportunity.short,
 			description: opportunity.description,
-			earn_format_mnoney: CoreServerHelpers.formatMoney(opportunity.opportunityTypeCd === 'sprint-with-us' ? opportunity.phases.aggregate.target : opportunity.earn, 2),
 			earn: CoreServerHelpers.formatMoney(opportunity.opportunityTypeCd === 'sprint-with-us' ? opportunity.phases.aggregate.target : opportunity.earn, 2),
```
Comment (SteveChapmanBCDX) [inline modules/opportunities/server/controllers/OpportunitiesServerController.ts]: Is this deleted line related to this fix, or just general cleanup of dead code?

File: modules/opportunities/server/controllers/OpportunitiesServerController.ts [hunk 3]
```diff
@@ -854,15 +850,11 @@ class OpportunitiesServerController {
 
 	private setMessageData = opportunity => {
 		opportunity.path = '/opportunities/' + (opportunity.opportunityTypeCd === 'sprint-with-us' ? 'swu' : 'cwu') + '/' + opportunity.code;
-		opportunity.earn_format_mnoney = CoreServerHelpers.formatMoney(opportunity.opportunityTypeCd === 'sprint-with-us' ? opportunity.phases.aggregate.target : opportunity.earn, 2);
 		opportunity.earn = opportunity.opportunityTypeCd === 'sprint-with-us' ? opportunity.phases.aggregate.target : opportunity.earn;
```
Comment (SteveChapmanBCDX) [inline modules/opportunities/server/controllers/OpportunitiesServerController.ts]: Same question.

File: modules/opportunities/server/controllers/OpportunitiesServerController.ts (hunk 2 above)
Comment (sutherlanda) [reply modules/opportunities/server/controllers/OpportunitiesServerController.ts]: Clean-up of dead code

File: modules/opportunities/server/controllers/OpportunitiesServerController.ts (hunk 3 above)
Comment (sutherlanda) [reply modules/opportunities/server/controllers/OpportunitiesServerController.ts]: Same answer :)

**Pull request closed without merging**